cypress_test/integration_tests/desktop/impress/slide_navigation_spec.js

```diff
@@ -0,0 +1,41 @@
+/* -*- js-indent-level: 8 -*- */
+/* global describe it cy require expect beforeEach*/
+
+var helper = require('../../common/helper');
+
+describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Slide Navigation', function() {
+
+    beforeEach(function() {
+        helper.setupAndLoadDocument('impress/slide_navigation.odp');
+    });
+
+    it('Arrow navigation', function() {
+        cy.cGet('#preview-img-part-0').click();
+
+        cy.cGet('#preview-img-part-0').type('{downarrow}');
+        cy.wait(500);
+        cy.cGet('#preview-frame-part-1').then(function(preview) {
```

Comment:
`wait` and `then` - should be avoided at any cost :)
It's a receipe for a race: https://docs.cypress.io/api/commands/should#Differences
Which was our biggest CI problem in recent months.

Reply:
avoiding wait in this case if a bit difficult, we have to wait for graphicselection message for make sure we get the correct results.

Reply:
No problem with just wait.
When we used wait + then - it can easily fail as it will not retry. using regular cy.cGet it will.
Thanks for fix :)